src/Neo/SmartContract/Helper.cs

```diff
@@ -361,5 +361,22 @@ internal static bool VerifyWitness(this IVerifiable verifiable, ProtocolSettings
             }
             return true;
         }
+
+        /// <summary>
+        /// Emits an <see cref="Instruction"/> with <see cref="OpCode.SYSCALL"/> System.Contract.Call with the specified parameters.
```

Comment:
Maybe introducing the `EmitXxxxxxx` to the Neo.VM.Helper is better because all other `Emit`s are there. Further more, do we really need this in other places? Could we only add this helper to tests related part?

Add if it's needed, it's better to tell users that they should place the parameters for `OpCode.SYSCALL` on the top of the stack before they use this Emit.

Reply:
> Maybe introducing the EmitXxxxxxx to the Neo.VM.Helper is better

It is, but I'd like to avoid cyclic dependencies between NeoVM and Neo core (this helper requires callflag argument).

>  Further more, do we really need this in other places?

Since we're not going to implement Koblitz witness builder in the core, the only user of this helper is testing code, so you're right.

> Could we only add this helper to tests related part?

Fixed, take a look at the updated code.